package/MDAnalysis/coordinates/base.py

```diff
@@ -141,9 +141,15 @@
 from ..auxiliary.core import get_auxreader_for
 
 class Namespace(object):
-    """Empty class 
-    To allow storing attributes in new namespace """
-    pass
+    """Class to allow storing attributes in new namespace. """
```

Comment:
Yes pandas gets a special treatment but that only allows the dictionary part to work with TAB completion as well. For normal dicts `d['key']` can be TAB completed in ipython. This also works for pandas and is hard coded into ipython. The normal completion from the members `__dict__` works for all classes. When looking for pandas I mean it's use that members can be accessed with `__getitem__` and `__getattrib__` at the same time.